content/docs/next/tutorial/introduction.md

```diff
@@ -17,11 +17,11 @@ On Unix based systems, download our tar.gz package.
 
 ```bash
 cd where/we/want/iofog-demo
-curl -L -o demo.tar.gz https://github.com/eclipse-iofog/demo/archive/v1.0.0.tar.gz
+curl -L -o demo.tar.gz https://github.com/eclipse-iofog/demo/archive/v1.2.0.tar.gz
 tar -zxvf demo.tar.gz --strip-components=1
 ```
 
-On windows, [download tutorial .zip package](https://github.com/eclipse-iofog/demo/archive/v1.0.0.zip). Then unzip the contents into a preferred working directory.
+On windows, [download tutorial .zip package](https://github.com/eclipse-iofog/demo/archive/v1.2.0.zip). Then unzip the contents into a preferred working directory.
```

Comment:
Can we look into(not now, maybe later, making this a dynamic link since it won't be linked under a certain version? If the file location isn't going to change, we could have it compared against the doc versions, and pick the latest published one to replace the v1.2.0 section?